cloud/terraform/sample/resources.json

```diff
@@ -0,0 +1,12 @@
+{
+    "provider": "aws",
+    "instances": [
+        {
+            "ami": "ami-0767af0854a146e3e",
+            "region": "us-east-1",
+            "instance_type": "",
+            "username": "ec2-user",
+            "name": "example-name-RHEL-8.6"
```

Comment:
Call me picky, but it sounds more natural "sample-name". If not, it would have to be "name-example" I think.